pyproject.toml

```diff
@@ -1,70 +1,60 @@
-[build-system]
-requires = ["setuptools >= 65", "setuptools-scm >= 7"]
-build-backend = "setuptools.build_meta"
-
-[project]
-name = "asyncio_mqtt"
-description = "Idiomatic asyncio wrapper around paho-mqtt"
+[tool.poetry]
+name = "aiomqtt-test"
+version = "0.0.0"  # Placeholder: Managed by poetry-dynamic-versioning
+description = "The idiomatic asyncio MQTT client, wrapped around paho-mqtt"
+license = "BSD-3-Clause"
+authors = ["Frederik Aalund <fpa@sbtinstruments.com>"]
+maintainers = ["Felix Böhm <felix@felixboehm.dev>", "Jonathan Plasse <jonathan.plasse@live.fr>"]
 readme = "README.md"
-authors = [{ name = "Frederik Aalund", email = "fpa@sbtinstruments.com" }]
-maintainers = [
-    { name = "Jonathan Plasse", email = "jonathan.plasse@live.fr" },
-    { name = "Felix Böhm", email = "felix@felixboehm.dev" },
-]
-keywords = ["mqtt", "async", "asyncio", "paho-mqtt", "wrapper"]
-license = { text = "BSD 3-Clause License" }
+packages = [{include = "asyncio_mqtt"}]
+repository = "https://github.com/sbtinstruments/asyncio-mqtt"
+documentation = "https://sbtinstruments.github.io/asyncio-mqtt"
+keywords = ["mqtt", "iot", "internet-of-things", "asyncio", "paho-mqtt", "mqttv5"]
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Intended Audience :: Developers",
-    "License :: OSI Approved",
-    "Operating System :: POSIX :: Linux",
-    "Operating System :: Microsoft :: Windows",
-    "Operating System :: MacOS",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Python :: Implementation :: PyPy",
```

Comment:
Why remove these classifiers?

Reply:
Poetry adds those automatically from the version number! 🙂

Reply:
Wow, awesome !